controllers/apicast_controller.go

```diff
@@ -63,8 +61,8 @@ func (r *ApicastReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 
 	// Apply defaults for reconcile but do not store them in the API
 	instance.Default()
-	json, _ := json.Marshal(instance.Spec)
-	log.V(1).Info("Apply defaults before resolving templates", "JSON", string(json))
+	// json, _ := json.Marshal(instance.Spec)
```

Comment:
intended comment?

Reply:
Oh yes, that should be deleted. This was logging all the custom resource with all default values applied but it was making the controller logs too noisy and was not very relevant information.

Reply:
Deleted f8dbed6